jarviscli/plugins/spinthewheel.py

```diff
@@ -0,0 +1,20 @@
+from plugin import plugin
+import random
+'''This code picks one of the random inputs given by the user
```

Comment:
Maybe make this comment a doc-String: Something like

```
@plugin("Spin wheel")
def spin(jarvis, s):
    """"
    This code picks (...)
    """"
    jarvis.say(...
```

This this text is printed when asked ``help spin wheel``.

Reply:
Thanks and I will look into it